### DISCUSSION TITLE
Roadmap for Traces

### DISCUSSION THREAD

**Discussion by peaceiris** [pipe-cd/pipecd#5166] [Ideas]
I observed that a feature related to Deployment Traces was added in the following issue. I’m convinced that there are other necessary features related to Traces for PipeCD. I’m curious about the future roadmap for Traces features in PipeCD. Is there any place where this is summarized?

- https://github.com/pipe-cd/pipecd/issues/4977

**Comment by Warashi** [comment]
There is no other roadmap except this RFC.
https://github.com/pipe-cd/pipecd/blob/master/docs/rfcs/0013-export-deployment-traces-with-otlp.md

> I’m convinced that there are other necessary features related to Traces for PipeCD.

I want to know the features you need. I would like you to tell me.
